docs/UseCases.md

```diff
@@ -0,0 +1,47 @@
+## About Professor M's School for Gifted Coders
+Professor M aka Professor Marum has created an exclusive not-for-profit school for gifted coders. Coders are a subspecies of human who are born with superhuman capabilities in technology and analysis. 
+
+Professor M is gathering the best and brightest to train them to do battle against the forces of evil (specifically bad customer experiences and poor project implementations). It is hazardous and demanding work. 
+
+Professor M has deployed Sugar as a platform to help him manage both students and faculty so he can focus on teaching.
+
+##High Level Use Cases
+
+The Professor M School uses Sugar for the following high level use cases:
+- Managing applicants, current students, former students, and professors
+- Tracking super groups
+- Soliciting donations from alumni and alumni affiliated super groups
+
+###Data Model
+
+| Professor M Module | Original Sugar Module | Description |
+| :--- | :--- | :---- |
+| Super Groups | Accounts | Super Groups are organizations of affiliated coders. |
+| Applicants | Leads | Applicants are prospective students that need to be accepted or rejected |
+| Students | Contacts | Students are coders who have been admitted to the Professor M School |
+| Donations | Opportunities | Donations are opportunities to raise funds as charitable contributions |
+| Funding Line Items | Revenue Line Items| Charitable contributions broken down by line item. |
+
+| Professor M Module | Custom Module Type |  Description |
+| :--- | :--- | :--- |
+| Professors | Person | School staff that helps Professor M teach student coders |
+
+### Implemented Use Cases
+
+#### Student Vital Tracking
+
+**Background**
+
+When doing battle against the forces of evil, it is possible that a coder is injured, comatose, or even (temporarily) killed. Professor M needs to report on the cause of death of any coder who has died in order to prevent the same horror from happening again.
+
+**Implementation Details**
+
+A custom dropdown field `vital_c` was added to the Students module and the Record view. The following vital statuses are selectable.
+    
+    'active' => 'Active'
+    'injured' => 'Injured'
+    'comatose' => 'Comatose'
+    'deceased' => 'Deceased'
+
+When `deceased` is selected, a new Death Information panel appears on Record view and users are required to enter a `cause_of_death_c`. This field is reportable.
```

Comment:
I tested this out, and the panel appears and disappears well.  The only thing that surprised me is that if I mark a student as deceased, input their cause of death, save, update the student to be active, save, and update the student to be deceased, the original cause of death is displayed.  You could make a case for the original cause of death being stored or not.  My bigger concern is that if I were to mark a student as deceased, change back to active, and pull the data out of a report, search, or the db, is it possible a student could be active with a cause of death?